cogstat/cogstat_stat.py

```diff
@@ -750,6 +804,68 @@ def variable_pair_standard_effect_size(data, meas_lev, sample=True, normality=No
     return standardized_effect_size_result
 
 
+def ordered_logistic_regression_variable_pair_standard_effect_size(data, meas_lev, sample=True):
+    """Calculate standardized effect size measures.
+    (Some stats are also calculated elsewhere, making the analysis slower, but separation is a priority.)
+
+    Parameters
+    ----------
+    data : pandas dataframe
+    meas_lev : str
+        Measurement level of variables.
+    sample : bool
+        True for sample descriptives, False for population estimations.
+
+
+    Returns
+    -------
+    html text
+    """
+    standardized_effect_size_result = ''
+    missing_pdf_result = False
+    if sample:
+        pdf_result = pd.DataFrame()
+        if meas_lev in ['int', 'unk']:
```

Comment:
If the function handles logistic regression, then aren't int and nom variables unnecessary?